app/src/main/java/org/novinger/jason/demoreader/datamodels/Article.java

```diff
@@ -0,0 +1,67 @@
+package org.novinger.jason.demoreader.datamodels;
+
+/**
+ * Created by jason on 4/27/16.
+ */
+public class Article {
+    private String id;
+    private int realId;
+    private String headline;
+    private String slug;
+    private Author[] authors;
+    private String summary;
+    private String url;
+
+    public Article() {}
+
+    public Article(String id, String headline, String slug, Author[] authors, String url, String summary) {
+        this.id = id;
```

Comment:
i personally never use `this.parameter = parameter` - i find it a little opaque.

while this guide is for _contributors_, not just anyone writing android, we used it at the Dot, and we use it at O'Reilly, and i use it personally: https://source.android.com/source/code-style.html